test/unit/nxos/mock_data/show_ip_interface_multiple_ips.json

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
Most of the info in this file is unused for this method.  Should the excess info be removed, or should we leave this in since it's realistic data?